docs/mpc/mpc.rst

```diff
@@ -29,68 +42,293 @@ An example of an exact match:
+    2018-08-18 11:00:00.000 97.17999999999998 23.724166666666665 3.482 2.914       48.6  15.1 12.9         52.76      92.4               --        --
+    2018-08-18 12:00:00.000 97.19583333333333 23.723611111111108 3.481 2.914       48.7  15.1 12.9         52.76      92.4               --        --
+    2018-08-18 13:00:00.000 97.21208333333333 23.723055555555554 3.481 2.914       48.7  15.1 12.9         52.75      92.4               --        --
+    2018-08-18 14:00:00.000 97.22791666666666  23.72222222222222  3.48 2.914       48.7  15.1 12.9         52.74      92.4               --        --
+    Length = 49 rows
+
+Start dates are parsed with Astropy's `~astropy.time.Time`.  For a
+weekly ephemeris in 2020:
+
+.. code-bock:: python
+
+    >>> eph = MPC.get_ephemeris('24', start='2020-01-01', step='7d', number=52)
+    >>> print(eph)
+              Date                  RA                 Dec         Delta   r   Elongation Phase  V   Proper motion Direction Uncertainty 3sig Unc. P.A.
+                                   deg                 deg           AU    AU     deg      deg  mag    arcsec / h     deg         arcsec         deg   
+    ----------------------- ------------------ ------------------- ----- ----- ---------- ----- ---- ------------- --------- ---------------- ---------
+    2020-01-01 00:00:00.000 209.16749999999996  -11.63361111111111 3.066 2.856       68.5  18.7 12.7         45.15     110.6               --        --
+    2020-01-08 00:00:00.000 211.11999999999995 -12.342500000000001  2.98 2.863       73.6  19.2 12.7         42.09     110.2               --        --
+    2020-01-15 00:00:00.000 212.93749999999997 -12.987222222222222 2.892  2.87       78.9  19.7 12.6          38.7     109.8               --        --
+    2020-01-22 00:00:00.000 214.60083333333333 -13.564722222222223 2.803 2.877       84.3  19.9 12.6         34.89     109.5               --        --
+                        ...                ...                 ...   ...   ...        ...   ...  ...           ...       ...              ...       ...
+    2020-12-02 00:00:00.000 252.88041666666666  -22.87638888888889 4.224 3.242        4.3   1.3 12.9         54.42      96.9               --        --
+    2020-12-09 00:00:00.000 255.62041666666664 -23.159166666666664 4.235  3.25        0.5   0.1 12.8         54.31      95.9               --        --
+    2020-12-16 00:00:00.000 258.36208333333326             -23.395 4.237 3.258        4.9   1.5 13.0         54.07      94.8               --        --
+    2020-12-23 00:00:00.000 261.09624999999994 -23.583055555555557 4.232 3.265        9.5   2.8 13.1         53.67      93.8               --        --
+    Length = 52 rows
+
+
+Observer location
+^^^^^^^^^^^^^^^^^
+
+Ephemerides may be calculated for Earth-based observers.  To calculate
+Makemake's ephemeris for the Discovery Channel Telescope (IAU
+observatory code G37):
+
+    >>> eph = MPC.get_ephemeris('Makemake', location='G37')
```

Comment:
This needs the `.. code-block:: python` directive

Reply:
Got it.